chat-plugins/wtp.js

```diff
@@ -0,0 +1,101 @@
+'use strict';
+/********************
+ * Who's that Pokemon?
+********************/
+const cards = require('../config/card-data.js');
+
+
+const maxMistakes = 6;
+
+class Wtp {
+	constructor(room, user, mon, hint1, hint2, hint3) {
+		this.room = room;
+		this.creator = user.userid;
```

Comment:
imo, make <code>this.creator</code> hold a reference to the user object instead of holding the user's userid